.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "docs/themes/docsy"]
 	path = docs/themes/docsy
-	url = git@github.com:google/docsy.git
+	url = https://github.com/google/docsy.git
```

Comment:
I don't think you should push this modifiction

Reply:
git@github.com bombs for me when building in CodeBuild (AWS) with an issue with key not present and access denied, so that's not cool ...

Reply:
you can keep your local modifications, it's just that this file is not linked to your pull request so... doesn't make sense to modify it :D